resources/assets/uad_lists.json

```
@@ -14060,7 +14060,7 @@
   {
     "id": "com.contextlogic.wish",
     "list": "Misc",
-    "description": "Wish Shopping (https://play.google.com/store/apps/details?id=com.contextlogic.wish)\nYou should not buy with Wish if you want my opinion.\n",
+    "description": "Wish Shopping (https://play.google.com/store/apps/details?id=com.contextlogic.wish)\nYou should not buy from Wish if you want my opinion.\n",
```

```suggestion
    "description": "Wish Shopping (https://play.google.com/store/apps/details?id=com.contextlogic.wish)",
```
It's up to the user whether he or she wants to use the website. There's no room for opinions in something objective like packages.